### ISSUE TITLE
config.cson中无法添加命令

### ISSUE THREAD

**Issue by yuhaoxin** [skywind3000/atom-shell-commands#14]
我把说明文档提供的compile指令复制到config.cson中，保存后会自动变成"atom-shell-commands": {}
如下图所示：
保存前
<img width="378" alt="1" src="https://cloud.githubusercontent.com/assets/11943130/20561128/d8286ae8-b1b7-11e6-9380-63efb3f6e4f9.PNG">
保存后
![image](https://cloud.githubusercontent.com/assets/11943130/20561141/eb054550-b1b7-11e6-9052-0089aab6f937.png)
请问如何解决，谢谢。

**Comment by skywind3000** [comment]
注意 indent, tab设置，要和cson其他行保持一致。

**Comment by AWPelican** [comment]
I'm experiencing the same problem as you're describing. I put in the settings exactly like you're saying, save the file, and then in a few moments get an empty set of curly braces (i.e., my settings are wiped out). I'm not sure if you found the solution or not, but I'd love to hear if you did!

This extension looks fantastic. I love the simplicity and power it would add to my editing environment. Hopefully I can end up getting it working.

**Comment by madakk** [comment]
I'm also experiencing this issue, my indentation all seem to be correct and everything is in order, but as soon as I save the config the contents get stripped and I'm left with empty curly braces. Anyone has any ideas on this?

**Comment by jpenca** [comment]
+1 exactly the same issue. I'm new to Atom and installing this package was the first thing I tried to do. no idea at all what's going on.

**Comment by madakk** [comment]
I had a short discussion about this on the Atom discussion forums, with the conclusion that this is definitely the package itself and not Atom causing this. There's also a good alternative suggested as part of te discussion.

https://discuss.atom.io/t/config-file-problem-related-to-atom-shell-commands/37338

**Comment by skywind3000** [comment]
你们不要在 config.cson里面写空的东西，配置好再保存注意indent，不然错误的东西或者空的东西一保存就会被删除掉

**Comment by madakk** [comment]
Look at the screenshot from @yuhaoxin, he's pasting the example code from the README as is. What's wrong with that?

**Comment by lemongrassnginger** [comment]
yup, I'm getting the same issue, trying to adjust the sample config from the readme. Is there some kind of config parsing that's not working? Is the sample config in the readme valid cson? 
I love the concept this package offers, would be awesome to get it working

**Comment by kedvsk** [comment]
My solution was to remove the .../.atom/config.cson file, restart atom and then edit the regenerated file with an external  editor. The file has to be stored as a UNIX file, not with UTF-8 or some else encoding.

But now my problem is, that the records within "atom-shell-commands" are not recognized, i.e. I don't have any idea how to use a configured command.

![image](https://cloud.githubusercontent.com/assets/22676820/22013407/27bf4b60-dc99-11e6-8de4-17fd507c0ea1.png)

**Comment by AWPelican** [comment]
This is a duplicate of issue #10, where using an external editor solved the issue for me. After using an external editor for the first command I created, I could edit config.cson in Atom as normal and not have my future changes wiped out. It was just that first time that I had to do it in an external editor.

**Comment by skywind3000** [comment]
Sorry, for replying late,
This repository is looking for maintainer now, https://github.com/skywind3000/atom-shell-commands/issues/20

**Comment by pinkie-ljz** [comment]
您好，我也遇到了题主一样的问题，一改变保存就自动归为{}。后面尝试了用其他的编辑器打开也是一样的。请问这个具体应该如何配置啊？折腾N久了。谢谢
